docs/flow/modin/core/dataframe/base/index.rst

```diff
@@ -0,0 +1,13 @@
+Modin BaseDataframe Interface
+=============================
+
+* :doc:`BaseDataframeAxisPartition <partitioning/axis_partition>` is an interface for a Core Dataframe object, representing a joined group of partitions along some axis (either rows or labels).
```

Comment:
it appeared that `BaseDataframeAxisPartition` implements some of the methods and defines some class fields, so it should be considered as an abstract class, right?

changed "is an interface" -> "is an abstract class"

Reply:
Those methods and class fields are supposed to be moved to a child class. I was confused about this phrase `an interface for a Core Dataframe object` instead of `an abstract class for a Core Dataframe AxisPartition object`. But now the whole sentence looks good too.